src/style.css

```diff
@@ -0,0 +1,123 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+body {
+  margin: 0;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 404px;
+  border: 1px solid;
+  border-color: rgba(243, 243, 243, 1);
+  border-radius: 5px;
+}
+
+.product-card__photo {
+  margin: 32px 20px;
+  margin-bottom: 40px;
+}
+
+.photo {
+  width: 160px;
+  height: 134px;
+}
+
+.product-card__model-name,
+.product-card__model-code,
+.product-card__reviews,
+.product-card__price,
+.product-card__button {
+  margin-left: 16px;
+  margin-right: 16px;
+}
+
+.product-card__model-name {
+  margin-bottom: 2px;
+}
+
+.model-name {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 150%;
+  color: #060b35;
+}
+
+.product-card__model-code,
+.product-card__price {
+  margin-bottom: 16px;
+}
+
+.model-code,
+.text-reviews {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 140%;
+  color: rgba(97, 96, 112, 1);
+}
+
+.product-card__reviews {
+  margin-bottom: 24px;
+}
+
+.reviews,
+.stars,
+.price {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-size: 16px 16px;
+  margin-right: 4px;
+}
+
+.stars :not(:nth-child(n+5)) {
+  background: url("images/star-active.svg") no-repeat;
+  background-size: 16px 16px;

```

Comment:
You don't need to set background-size: 16px 16px; Use other background qualities,
 if you need 
of course